x-pack/docs/en/security/authorization/built-in-roles.asciidoc

```diff
@@ -72,10 +72,12 @@ NOTE: This role does *not* provide the ability to create indices; those privileg
 must be defined in a separate role.
 
 [[built-in-roles-kibana-dashboard]] `kibana_dashboard_only_user` ::
-Grants access to the {kib} Dashboard and read-only permissions to Kibana.
-This role does not have access to editing tools in {kib}. For more
-information, see
-{kibana-ref}/xpack-dashboard-only-mode.html[{kib} Dashboard Only Mode].
+(This role is deprecated, please use 
+{kibana-ref}/kibana-privileges.html#kibana-feature-privileges[{kib} feature privileges]
+instead).
+Grants access to the {kib} Dashboard and read-only permissions to every 
+{kibana-ref}/xpack-spaces.html[Space in {kib}].
+This role does not have access to editing tools in {kib}. 
```

Comment:
^^ @legrego Can you review this change?

Reply:
Looks correct to me. I think something like this may be clearer:

> Grants read-only access to the {kib} Dashboard application within every 
{kibana-ref}/xpack-spaces.html[Space in {kib}].